website_event_sale_check_order_before_payment/static/src/js/website_sale_payment.js

```diff
@@ -0,0 +1,17 @@
+odoo.define("website_event_sale_check_order_before_payment.payment", function(require) {
+    "use strict";
+
+    const publicWidget = require("web.public.widget");
+
+    publicWidget.registry.WebsiteSalePayment.include({
+        /**
+         * @override
+         */
+        start: function() {
+            const res = this._super.apply(this, arguments);
+            this.$payButton = $("button#o_payment_form_check_order_and_pay");
```

Comment:
What's this for? 🤔 
Maybe it's worth adding a docstring here

Reply:
Yep, I'll add one :)
The standard button is enabled/disabled according whether the cgv checkbox is checked, and all the logic is done based on this `$payButton`. Overriding the content of the attribute allows to keep the standard behavior related to cgv_checkbox on the button with the new ID

Reply:
Alright then ;) thanks!